common/params.cc

```diff
@@ -89,6 +89,7 @@ class FileLock {
 
 std::unordered_map<std::string, uint32_t> keys = {
     {"AccessToken", CLEAR_ON_MANAGER_START | DONT_LOG},
+    {"Adb", PERSISTENT},
```

Comment:
for consistently with SSH:
```suggestion
    {"AdbEnabled", PERSISTENT},
```

Reply:
Sure thats all good